cliquet/cache/__init__.py

```diff
@@ -21,7 +29,14 @@ def ping(self):
         :returns: `True` is everything is ok, `False` otherwise.
         :rtype: boolean
         """
-        raise NotImplementedError
+        try:
+            if random.random() > _HEARTBEAT_DELETE_RATE:
```

Comment:
``random.random() < _HEARTBEAT_DELETE_RATE``

In case of 0.5 it is the same but not with 0.3